tfe.go

```diff
@@ -743,3 +747,22 @@ func checkResponseCode(r *http.Response) error {
 
 	return fmt.Errorf(strings.Join(errs, "\n"))
 }
+
+func readFile(path string) (*bytes.Buffer, error) {
```

Comment:
`readFile` doesn't feel right; Can we convey that we're actually packing up a file/path somehow? `packContents(path string)`, something? 🤷🏻‍♂️

Reply:
Yeah, this felt off but wasn't sure on the right naming. `packContents` could work. 🤔